packages/dai-plugin-migrations/src/migrations/GlobalSettlementCollateralClaims.js

```diff
@@ -0,0 +1,47 @@
+import BigNumber from 'bignumber.js';
+import { RAY } from '../constants';
+
+export default class GlobalSettlementCollateralClaims {
+  constructor(manager) {
```

Comment:
might be a good idea to rename this from `manager` since when i first saw it i thought you might be passing an instance of ServiceManager instead of the service itself

Reply:
Renamed it to the really generic `container`, because it's pretty much a container for external services. Alternatively could call it something like 'serviceProvider'?